test/uk/gov/hmrc/pillar2stubs/controllers/ObligationsAndSubmissionsControllerSpec.scala

```diff
@@ -368,10 +361,310 @@ class ObligationsAndSubmissionsControllerSpec
     ) { (id, expectedResponse) =>
       val result = route(app, request(id)).value
 
-      status(result) shouldEqual 200
-      contentAsJson(result) shouldBe Json.toJson(expectedResponse)
+      status(result).shouldEqual(200)
+      contentAsJson(result).shouldBe(Json.toJson(expectedResponse))
     }
+  }
+
+  test("Returns single active accounting period with no submission when Pillar2-Id is XEPLR9999999991") {
+    implicit val pillar2Id: String = "XEPLR9999999991"
```

Comment:
This does not seem to be part of the Scala 3 migration ticket. Why are we making this change?